### PR TITLE
Add deprecation notive for Hub V1 API

### PR DESCRIPTION
**Pull request by silvin-lubecki**
### Proposed changes

Added a deprecated section to the Hub API, related to futur Hub API V1 deprecation.

### CODE REVIEW

File: docker-hub/api/deprecated.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+description: Docker Hub API v1 (deprecated)
+keywords: kitematic, deprecated
+title: Docker Hub API v1 (deprecated)
+---
+
+> **Deprecated**
+>
+> Docker Hub API v1 has been deprecated. Please use Docker Hub API v1 instead.
```
Comment (usha-mandya) [inline docker-hub/api/deprecated.md:9]: @silvin-lubecki Should this be v2 instead?

Comment (silvin-lubecki) [reply docker-hub/api/deprecated.md:9]: Of course... 😅

Comment (silvin-lubecki) [reply docker-hub/api/deprecated.md]: Fixed https://github.com/docker/docker.github.io/pull/15461/commits/0f400bf78ba6571c338fa7fd89111a55d50861ad

**Comment by silvin-lubecki** [comment]
@usha-mandya PTAL 🙏

**Review by jerae-duffin — APPROVED** [review]
LGTM!

**Comment by jerae-duffin** [comment]
DO NOT MERGE. Waiting for Silvin to confirm the updates from his team.

**Pull request merged**